src/debug/ee/i386/dbghelpers.S

```diff
@@ -8,7 +8,11 @@
 //extern FuncEvalHijackWorker:proc
 
 // @dbgtodo- once we port Funceval, use the ExceptionHijack stub instead of this func-eval stub.
+#ifdef WIN64EXCEPTIONS
 NESTED_ENTRY FuncEvalHijack, _TEXT, UnhandledExceptionHandlerUnix
```

Comment:
I would prefer keeping it here as is and adding a dummy UnhandledExceptionHandlerUnix to unixstubs.cpp.